docs/install-run/special/windows.txt

```diff
@@ -38,13 +40,13 @@ Allow the ports CrateDB uses through the Windows Firewall:
 Install CrateDB
 ===============
 
-:doc:`Download the CrateDB Tarball <tarball>`, expand it and move to a
+:ref:`Download the CrateDB Tarball <basic-install>`, expand it and move to a
```

Comment:
shouldn't this refer to  <install_targz> instead?

Reply:
@seut that is no longer a valid reference. there is now only one page that documents the tarball install method, and that is `basic-install` in the getting started docs

Reply:
@nslater hm and whats about that? https://github.com/crate/getting-started/pull/17/files#diff-60a5e6172bb738429e778d057d301612R35

Reply:
@seut you're right! I was confused! sorry! thanks! pushed a fixup